### ISSUE TITLE
A13 1.9.6 does not work

### ISSUE THREAD

**Issue by zinhuofc** [akhilkedia/AllTrans#59]
focus for reddit:https://play.google.com/store/apps/details?id=allen.town.focus.reddit&hl=en_US
infinity: https://f-droid.org/pt_BR/packages/ml.docilealligator.infinityforreddit/
A13
[mylogcat.txt](https://github.com/akhilkedia/AllTrans/files/12205388/mylogcat.txt)
translation with google does not work
observation the com.android.providers.settings file is not marked in lsposed

**Comment by zinhuofc** [comment]
[azure.txt](https://github.com/akhilkedia/AllTrans/files/12205443/azure.txt)
with azure it didn't work either

**Comment by wzgrx** [comment]
Android 12 is also disabled and cannot be used

**Comment by rushiranpise** [comment]
@akhilkedia maybe let us know the lsposed version you are using
And the app to be translated

Currently im using latest lsposed ci version 1.8.6 v6951 and its not working with apps i test
for eg
en to hi or cn to en

**Comment by akhilkedia** [comment]
Thanks for the log.
The issue seems to be that AllTrans is not detecting your settings, and is hence not doing any translation.
The issue seems to be related to some other people reporting that the "com.android.providers.settings" becomes unchecked is LSPosed - is this the same for you as well?

> [azure.txt](https://github.com/akhilkedia/AllTrans/files/12205443/azure.txt) with azure it didn't work either

**Comment by akhilkedia** [comment]
I am using LSPosed(1.8.6 v6951) via LSPatch version 0.5.1(378). I will need to investigate with newer LSPosed.

> @akhilkedia maybe let us know the lsposed version you are using And the app to be translated
> 
> Currently im using latest lsposed ci version 1.8.6 v6951 and its not working with apps i test for eg en to hi or cn to en
> 
> Device is running oneui 4.1 a12 with kernelsu

**Comment by wzgrx** [comment]
> Thanks for the log. The issue seems to be that AllTrans is not detecting your settings, and is hence not doing any translation. The issue seems to be related to some other people reporting that the "com.android.providers.settings" becomes unchecked is LSPosed - is this the same for you as well?
> 
> > [azure.txt](https://github.com/akhilkedia/AllTrans/files/12205443/azure.txt) with azure it didn't work either

Yes, if checked, it will automatically cancel, I don't know why. Also, do you have any updates to Google's offline translation package

**Comment by akhilkedia** [comment]
> > Thanks for the log. The issue seems to be that AllTrans is not detecting your settings, and is hence not doing any translation. The issue seems to be related to some other people reporting that the "com.android.providers.settings" becomes unchecked is LSPosed - is this the same for you as well?
> > > [azure.txt](https://github.com/akhilkedia/AllTrans/files/12205443/azure.txt) with azure it didn't work either
> 
> Yes, if checked, it will automatically cancel, I don't know why. Also, do you have any updates to Google's offline translation package

Thanks for confirming. I believe this is due to some change on LSPosed's side. I will check.
Regarding google's offline translation, they have updated models on the server side to be much much faster, if your device supports it.
Simply clearing cache of AllTrans, and downloading the translation models again (by selecting the to/from translation language) will download the new models. But given that AllTrans isn't working for you as of yet, you will have to wait to benefit from this..

**Comment by wzgrx** [comment]
> > > Thanks for the log. The issue seems to be that AllTrans is not detecting your settings, and is hence not doing any translation. The issue seems to be related to some other people reporting that the "com.android.providers.settings" becomes unchecked is LSPosed - is this the same for you as well?
> > > > [azure.txt](https://github.com/akhilkedia/AllTrans/files/12205443/azure.txt) with azure it didn't work either
> > 
> > 
> > Yes, if checked, it will automatically cancel, I don't know why. Also, do you have any updates to Google's offline translation package
> 
> Thanks for confirming. I believe this is due to some change on LSPosed's side. I will check. Regarding google's offline translation, they have updated models on the server side to be much much faster, if your device supports it. Simply clearing cache of AllTrans, and downloading the translation models again (by selecting the to/from translation language) will download the new models. But given that AllTrans isn't working for you as of yet, you will have to wait to benefit from this..

Okay, thank you very much

**Comment by zinhuofc** [comment]
> > > Obrigado pelo registro.  O problema parece ser que o AllTrans não está detectando suas configurações e, portanto, não está fazendo nenhuma tradução.  O problema parece estar relacionado a algumas outras pessoas relatando que o "com.android.providers.settings" fica desmarcado é LSPosed - isso é o mesmo para você também? 
> > > > [azure.txt ](https://github.com/akhilkedia/AllTrans/files/12205443/azure.txt) com azure também não funcionou
> > 
> > 
> > Sim, se marcado, cancelará automaticamente, não sei por quê.  Além disso, você tem alguma atualização para o pacote de tradução off-line do Google
> 
> Obrigado por confirmar. Acredito que isso se deva a alguma mudança do lado de LSPosed. Eu vou verificar. Em relação à tradução off-line do Google, eles atualizaram os modelos no lado do servidor para serem muito mais rápidos, se o seu dispositivo suportar. Basta limpar o cache do AllTrans e baixar os modelos de tradução novamente (selecionando o idioma de tradução de/para) para baixar os novos modelos. Mas como o AllTrans ainda não está funcionando para você, você terá que esperar para se beneficiar disso.



> 

😃

**Comment by zinhuofc** [comment]
> > > Thanks for the log. The issue seems to be that AllTrans is not detecting your settings, and is hence not doing any translation. The issue seems to be related to some other people reporting that the "com.android.providers.settings" becomes unchecked is LSPosed - is this the same for you as well?
> > > > [azure.txt](https://github.com/akhilkedia/AllTrans/files/12205443/azure.txt) with azure it didn't work either
> > 
> > 
> > Yes, if checked, it will automatically cancel, I don't know why. Also, do you have any updates to Google's offline translation package
> 
> Thanks for confirming. I believe this is due to some change on LSPosed's side. I will check. Regarding google's offline translation, they have updated models on the server side to be much much faster, if your device supports it. Simply clearing cache of AllTrans, and downloading the translation models again (by selecting the to/from translation language) will download the new models. But given that AllTrans isn't working for you as of yet, you will have to wait to benefit from this..

😃

**Comment by zinhuofc** [comment]
I don't know if it helps, but in previous versions of lsposed it keeps the settings checked and it still doesn't work

**Comment by xerel89** [comment]
For me is also not working [https://mega.nz/file/aHRhTSgZ#OJrbPu9LSnqjMh-5lbgJRrs8TejJYEzFx1wQHVS5hjg ](log here)

**Comment by zinhuofc** [comment]
I was so happy, now every day I come here to see if there's an update :/

**Comment by akhilkedia** [comment]
At the moment I don't have a device with Lsposed installed, so I am currently using LSPatch.
I tried using Lsposed on an emulator, but latest magisk requires a newer desktop operating system version that I am currently using, so I will have to update my PC OS for that.
Once I do that, I will get to testing AllTrans with LSPosed, but at this moment I have no way of doing so.

**Comment by rushiranpise** [comment]
> At the moment I don't have a device with Lsposed installed, so I am currently using LSPatch. I tried using Lsposed on an emulator, but latest magisk requires a newer desktop operating system version that I am currently using, so I will have to update my PC OS for that. Once I do that, I will get to testing AllTrans with LSPosed, but at this moment I have no way of doing so.

which emulator are you using?

try https://github.com/MustardChef/WSABuilds

**Comment by akhilkedia** [comment]
> which emulator are you using?
> 
> try https://github.com/MustardChef/WSABuilds

Ah, I was trying the official Android AVD Emulator.
But this seems promising, I'll try using this!

**Comment by zinhuofc** [comment]
> > which emulator are you using?
> > try https://github.com/MustardChef/WSABuilds
> 
> Ah, I was trying the official Android AVD Emulator. But this seems promising, I'll try using this!

😞😔🙁☹️

**Comment by privacyguy123** [comment]
> At the moment I don't have a device with Lsposed installed, so I am currently using LSPatch. I tried using Lsposed on an emulator, but latest magisk requires a newer desktop operating system version that I am currently using, so I will have to update my PC OS for that. Once I do that, I will get to testing AllTrans with LSPosed, but at this moment I have no way of doing so.

AllTrans does not work on Lsposed because of how the hooks work under the hood. The sooner you are able to test on Lsposed the better because this module is totally broken unless you use Lspatch.

**Comment by zinhuofc** [comment]
I'm sad to this day without updating to lsposed

**Comment by zinhuofc** [comment]
24/10 :/

**Comment by zinhuofc** [comment]
Come back dude and upgrade to lsposed, we need your app you promised :/

**Comment by pablovinicius332** [comment]
I can't even eat anymore 🥺 lsposed lsposed lsposed

**Comment by pablovinicius332** [comment]
> > At the moment I don't have a device with Lsposed installed, so I am currently using LSPatch. I tried using Lsposed on an emulator, but latest magisk requires a newer desktop operating system version that I am currently using, so I will have to update my PC OS for that. Once I do that, I will get to testing AllTrans with LSPosed, but at this moment I have no way of doing so.
> 
> AllTrans does not work on Lsposed because of how the hooks work under the hood. The sooner you are able to test on Lsposed the better because this module is totally broken unless you use Lspatch.

You seem to understand, we help man make a fork
